contracts/tools/DVP.sol

```diff
@@ -381,55 +433,58 @@ contract DVP is Ownable, ERC1820Client, IERC1400TokensRecipient, ERC1820Implemen
    * @param ethValue Value sent (only used for ETH)
    * @param erc1400TokenValue Value sent (only used for ERC1400)
    */
-  function _acceptTrade(uint256 index, address sender, uint256 ethValue, uint256 erc1400TokenValue) internal {
+  function _acceptTrade(uint256 index, address sender, uint256 ethValue, uint256 erc1400TokenValue, bytes32 preimage) internal {
     Trade storage trade = _trades[index];
     require(trade.state == State.Pending, "Trade is not pending");
 
+    address recipientHolder;
     address selectedHolder;
     if(sender == trade.holder1) {
       selectedHolder = trade.holder1;
+      recipientHolder = trade.holder2;
     } else if(sender == trade.holder2) {
       selectedHolder = trade.holder2;
+      recipientHolder = trade.holder1;
     } else if(trade.holder2 == address(0)) {
       trade.holder2 = sender;
       selectedHolder = trade.holder2;
```

Comment:
`recipientHolder` is undefined when `trade.holder2 == address(0)` ==> Is this not an issue when we check if `_holdExists`?

Reply:
Fixed bug, `recipientHolder` is now `trade.holder1`. I believe this is the correct behavior